charts/k8s-ovpn-chart/templates/network-policy.yaml

```diff
@@ -1,9 +1,8 @@
-{{- if .Values.limitTraficToNamespace -}}
+{{- if .Values.limitTraficToNamespace }}
 kind: NetworkPolicy
 apiVersion: networking.k8s.io/v1
 metadata:
   name: {{ include "k8s-ovpn-chart.fullname" . }}
-  namespace: {{ .Values.namespace.name }}
```

Comment:
Should the `NetworkPolicy` live in the `default` namespace? I'd argue it should be in the same one as the `Deployment`.

Reply:
updated